ansible/roles/ocp4-workload-mlops/tasks/workload.yml

```diff
@@ -129,9 +51,23 @@
   when: r_codeready_cr.resources | list | length == 0
   include_tasks: install-codeready.yaml
 
+- name: install argocd operator
```

Comment:
There is already a role for this. Can we use it instead of recreating things in here?

https://github.com/redhat-cop/agnosticd/tree/development/ansible/roles_ocp_workloads/ocp4_workload_argocd_operator